src/styles/components/_related-components.scss

```diff
@@ -1,3 +1,5 @@
+$breakpoint-small-tablet: 38.125rem; // 610px/16
```

Comment:
this bp probably shouldn't be technology based ("tablet") but function based ("move-right")

Reply:
I made another note to create a new card to rename more of our breakpoints! I think we talked about that last week too

https://github.com/sparkbox/accessible-components/pull/89/commits/d3aae7dedbb9362f058ee30fe4e6be5d0598c269